api/rosetta/status.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Alvalor S.A.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package rosetta
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+
+	"github.com/optakt/flow-dps/models/identifier"
+	"github.com/optakt/flow-dps/models/rosetta"
+)
+
+type StatusRequest struct {
+	NetworkID identifier.Network `json:"network_identifier"`
+}
+
+type StatusResponse struct {
+	CurrentBlockID        identifier.Block `json:"current_block_identifier"`
+	CurrentBlockTimestamp int64            `json:"current_block_timestamp"`
+	OldestBlockIdentifier identifier.Block `json:"oldest_block_identifier"`
```

Comment:
Will we (can we?) support any `sync_status` info?

Reply:
There is no such concept as a synchronization status for the Flow DPS. This is relevant in case it runs on top of so-called full nodes on other blockchain networks.